.travis/centos8/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos:latest
+
+RUN yum -y update
+
+RUN yum -y groupinstall 'Development Tools'
+RUN yum -y install dnf-utils --nogpgcheck
+RUN yum -y install wget --nogpgcheck
+RUN yum -y install sudo --nogpgcheck
+
+RUN dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+
+RUN yum config-manager --set-enabled PowerTools
+
+RUN wget http://mirror.centos.org/centos/7/os/x86_64/Packages/xorg-x11-server-source-1.20.4-7.el7.noarch.rpm
```

Comment:
I don't think that taking the xorg-x11-server-source rpm from el7 is the right thing to do